### DISCUSSION TITLE
Anime functionality

### DISCUSSION THREAD

**Discussion by snipeart007** [metafates/mangal#143] [Ideas]
Lead: @metafates

I recently started using Mangal and I'm loving it.

I have gone through the code and I have noticed something. The thing is mangal is very modular and extensible.

I think it's possible to create a anime downloader and player too, using the same tui, anilist, etc. functionality.

But, it would need a completely different project to make it work.

The anime community is much more bigger than manga. Anime lovers need the ability to change their download sources periodically as they either get shut down or become unusable due to the heavy traffic.

So, I recommend that you create a new scraper for anime, while using the functionality of mangal wherever possible.

I know this is going to take a while, and you may refrain yourself from starting this new project, but, if this gets successful, many anime lovers will shift to this one.

The beautiful design of mangal can even be used by the people who use mobile phones, as, it is very convienient to use.

A very famous website used by anime lovers, AniMixPlay, has also recently been shut down forever.

This means, even those who used AniMixPlay, will shift to thsi new project.

I hope you consider this idea.

By the way, I'm not a good programmer and only know the basics of Go. I'm just a genuine anime and manga lover, who wants anime to reach everyone in this free world of internet.

**Comment by metafates** [comment]
Hello!

Thanks for the suggestion,

I had the same thought in mind so for the last few months I've been making a new cross-platform app called `awirix` which will support any possible media scraping - anime, manga, movies, music even? All this due to well thought extensions structure.

I can say that it works pretty well already, but I will need some time before releasing it - there's too much stuff to document.
You can take a look at it here https://github.com/awirix/awirix

Name may change in the future

**Reply by snipeart007** [reply]
That's very nice to hear. I am excited to see what you come up with.
